.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             - ~/.cache/yarn
       - run:
           name: Build all packages with lerna
-          command: 'yarn build --since master'
+          command: 'yarn build --skip-nx-cache'
```

Comment:
Why skip nx cache?

Reply:
it is to make sure it will build all packages specially for a next release.